cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesGroupControllerTestIT.java

```diff
@@ -485,9 +489,9 @@ void test_add_assigned_locs() throws Exception {
             .body("office-id", equalTo(group.getOfficeId()))
             .body("id", equalTo(group.getId()))
             .body("description", equalTo(group.getDescription()))
-            .body("assigned-time-series[0].timeseries-id", equalTo(timeSeriesId2))
-            .body("assigned-time-series[0].alias-id", equalTo("AliasId2"))
-            .body("assigned-time-series[0].ref-ts-id", equalTo(timeSeriesId2));
+            .body("assigned-time-series[1].timeseries-id", equalTo(timeSeriesId2))
```

Comment:
changing the index 0->1 makes sense b/c the var is timeSeriesId2.  Should the test also be looking at index 0?

Reply:
Probably to verify it's the same, but I made that change because the asserted clearly showed it should be the 2nd.